Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +12,198 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0) return "N/A.";
+
+            int highestBalance = 0;
```

Comment:
With the function dealing with currency amounts is int the most appropriate type to represent an amount of currency?

Reply:
For currency amount it's better to use decimal